dredis/lua.py

```diff
@@ -46,16 +46,19 @@ def _convert_redis_types_to_lua_types(self, result):
         The implementation can be found at:
         https://github.com/antirez/redis/blob/5b4bec9d336655889641b134791dfdd2adc864cf/src/scripting.c#L106-L201
         """
+
         if isinstance(result, (tuple, list, set)):
             table = self._lua_runtime.table()
             for i, elem in enumerate(result, start=1):
+                print type(elem)
                 table[i] = self._convert_redis_types_to_lua_types(elem)
             return table
         elif result is None:
             return False
         elif result is True:
             return 1
-        elif isinstance(result, SimpleString):
+        # SimpleString was failing isinstance checks for strings, and also did not consider unicode
+        elif isinstance(result, (str, unicode)):
```

Comment:
@hltbra this change might require some review. All tests passed, but not sure if there are other implications.

Reply:
`SimpleString` is a special type, I am not sure this should change (there's a test failure, not sure if it's because of this).

Reply:
@hltbra I see, I am changing back to what we had and updating the tests to reflect this.